extensions/amp-bind/amp-bind.md

```diff
@@ -680,25 +731,38 @@ For example, if your `amp-state` element makes an XHR to an endpoint, while the
 
 #### Attributes
 
-<table>
-  <tr>
-    <td width="40%"><strong>src</strong></td>
-    <td><p>The URL of the remote endpoint that will return the JSON that will update this <code>amp-state</code>. This must be a CORS HTTP service.</p>
-<p>The <code>src</code> attribute allows all standard URL variable substitutions. See the <a href="../../spec/amp-var-substitutions.md">Substitutions Guide</a> for more info.</p>
-<p>{% call callout('Important', type='caution') %}
-  The endpoint must implement the requirements specified in the <a href="https://amp.dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests">CORS Requests in AMP</a> spec.
-  {% endcall %}</p></td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>credentials (optional)</strong></td>
-    <td><p>Defines a <code>credentials</code> option as specified by the <a href="https://fetch.spec.whatwg.org/">Fetch API</a>.</p>
-<ul>
-  <li>Supported values: `omit`, `include`</li>
-  <li>Default: `omit`</li>
-</ul>
-<p>To send credentials, pass the value of <code>include</code>. If this value is set, the response must follow the <a href="https://www.ampproject.org/docs/fundamentals/amp-cors-requests#cors-security-in-amp">AMP CORS security guidelines</a>.</p></td>
-  </tr>
-</table>
+[filter formats="websites, stories, ads"]
+
+##### src
+
+The URL of the remote endpoint that will return the JSON that will update this `amp-state`. This must be a CORS HTTP service. The `src` attribute allows all standard URL variable substitutions. See the [Substitutions Guide](../../spec/amp-var-substitutions.md) for more info.
+
+[tip type="important]
+The endpoint must implement the requirements specified in the [CORS Requests in AMP](https://amp.dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests) spec.
+[/tip]
+
+##### credentials (optional)
+
+Defines a `credentials` option as specified by the [Fetch API](https://fetch.spec.whatwg.org/).
+
+- Supported values: `omit`, `include`
+- Default: `omit`
+
+To send credentials, pass the value of `include`. If this value is set, the response must follow the [AMP CORS security guidelines](https://amp.dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests/#cors-security-in-amp).
+[/filter]
+
+[filter formats="email"]
+
+##### Invalid AMP email attributes
+
+Security and privacy disallow the use of the following attributes on the AMP email format.
```

Comment:
Should we leave out "security and privacy"? I think it just raises more questions than it answers.

Reply:
Agreed, updated. let me know if the wording works for you.